cosmos/operators/local.py

```diff
@@ -629,11 +675,17 @@ def get_openlineage_facets_on_complete(self, task_instance: TaskInstance) -> Ope
         )
 
     def build_and_run_cmd(
-        self, context: Context, cmd_flags: list[str] | None = None
+        self,
+        context: Context,
+        cmd_flags: list[str] | None = None,
+        run_as_async: bool = False,
```

Comment:
The `run_as_async` may sound misleading since we do not run query in this case, right?
shall we rename to `mock_query_execution` or something else?